container/run.sh

```diff
@@ -224,16 +226,16 @@ get_options() {
 	HF_CACHE=
     fi
 
-    if [ ! -z "$HF_CACHE" ]; then
-	mkdir -p $HF_CACHE
+    if [ ! -n "$HF_CACHE" ]; then
```

Comment:
Is this right? Should it be "-n" instead of "! -n"? It looks like -z was changed to -n but the `!` wasn't inverted.

Reply:
```suggestion
    if [ -n "$HF_CACHE" ]; then
```